public/site.html

```diff
@@ -9,8 +9,17 @@
             * Specify an Episode instead of an actual date
             * JQuery
     -->
-    <link href="main.css" , type="text/css" , rel=stylesheet />
-    <!--<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>-->
+
+    <!-- Our own CSS File-->
+    <link href="main.css" type="text/css" rel=stylesheet />
+
+    <!-- JQuery & JQuery UI from the Google Servers. Currently in use for picking dates across Browsers. -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>
```

Comment:
Adding 3 huge libraries (jQuery, jQuery UI and Modernizr) just for picking the date seems way too much. We already have D3.js, I'm sure there is a better alternative. Let's discuss during the meeting later.